app/workers/commit_monitor_handlers/commit_range/rubocop_checker.rb

```diff
@@ -19,15 +19,10 @@ def perform(branch_id, _new_commits)
   private
 
   def process_branch
-    diff_details = diff_details_for_merge
-
     unmerged_results = []
+    unmerged_results << Linter::Rubocop.new(branch).result
 
-    files = extract_ruby_files(diff_details)
-    if files.any?
-      unmerged_results << linter_results('rubocop', :format => 'json', nil => files)
-    end
-
+    diff_details = diff_details_for_merge
```

Comment:
I guess you don't really get any gains from this PR until the haml linter no longer needs to go to a git checkout either?